daycare/models.py

```diff
@@ -50,6 +50,12 @@ def create(email, username, password, name, description, price_per_day, capacity
 
         return new_daycare
 
+    def get_daycare_primary_image_url(self):
+        daycare_images = Image.get_images_by_daycare_id(daycare_id=self.id)
+        if daycare_images is not None and daycare_images.first() is not None:
+            return daycare_images.first().url
+        return "../../static/images/daycare-default-profile-image.jpeg"
+
 
```

Comment:
This code was implemented in #89 right? so I think it needs to be in another separate commit and add in the description on the PR that this PR depends on #89

Reply:
It is indicated in the description